rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1867,11 +1867,6 @@ void main_window::OnEmuStop()
 	ui->actionManage_Users->setEnabled(true);
 	ui->confCamerasAct->setEnabled(true);
 
-	if (std::exchange(m_sys_menu_opened, false))
```

Comment:
What? why? maybe I want to see how the game handles pause events for various reasons. This is not just an emulator aftet all it's also a debugger.

Reply:
How often have you clicked the button in the past 12 months?

Reply:
You make it impossible to get to specific code paths in PS3 application that would normally be accessible. If you dislike it being in the toolbar you should put it elsewhere.

Reply:
Also maybe with other optional values to send in a combobox.